### PR TITLE
Bump @oacore/stylelint-config-base to 2.0.0-stylelint.1

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
Drops Prettier support in the Stylelint.

Updates CSS files after rule changes in the stylelint config.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@babel/preset-react": "^7.10.4",
     "@oacore/eslint-config-react": "^1.0.6",
     "@oacore/prettier-config": "^1.0.6",
-    "@oacore/stylelint-config-base": "^1.0.6",
+    "@oacore/stylelint-config-base": "^2.0.0-stylelint",
```
Comment (Joozty) [inline package.json:24]: You should have kept just 2.0.0 without the suffix.

Comment (viktor-yakubiv) [reply package.json:24]: What's the problem with the prerelease tag?

Comment (Joozty) [reply package.json]: Nothing is wrong but now all @oaocore/configs packages will need to be upgraded to this version with prerelease tag and it doesn't look good to my eye 😄

Comment (viktor-yakubiv) [reply package.json]: Don't update then 😄

**Pull request merged**